### ISSUE TITLE
Error while Archiving the app.

### ISSUE THREAD

**Issue by kunass2** [AppLozic/Applozic-Chat-iOS-Framework#19]
>error: remove /Users/kuna/Library/Developer/Xcode/DerivedData/Chat-grlsepxoorbacsgwklisrppobsjy/Build/Intermediates/ArchiveIntermediates/Chat/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/Applozic.framework/hu.lproj: Directory not empty

What can I do to fix that issue?

**Comment by Abhishek9634** [comment]
Clear your derived data

**Comment by kunass2** [comment]
it worked. Thank you.

**Comment by jlubeck** [comment]
Mmm, every time that I switch between a simulator and an actual device I need to clear the derived data because of this error. I don´t have to do this with any other framework. Any chance this can be fixed? It´s kind of annoying... Thanks!

**Comment by Abhishek9634** [comment]
In the next release we will fix

**Comment by jlubeck** [comment]
Awesome!

**Comment by jlubeck** [comment]
This is still not working on 3.8.4. Wouldn't it be wise to keep it open until it's actually fixed?
Thanks!

**Comment by jlubeck** [comment]
Mmm, this is worse than I thought. Now I can't even make an Archive for sending to the App Store.
No matter how many times I delete my DerivedData folder, I always get these errors.

error: make directory /Users/jan/Library/Developer/Xcode/DerivedData/-fpubyzzpfryffqceorqoclebdggf/Build/Intermediates/ArchiveIntermediates/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/Applozic.framework/en.lproj: File exists

error: remove /Users/jan/Library/Developer/Xcode/DerivedData/-fpubyzzpfryffqceorqoclebdggf/Build/Intermediates/ArchiveIntermediates/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/Applozic.framework/en.lproj: Directory not empty

Not at the same time. Sometimes one, sometimes the other. With or without deleting the DerivedData folder.

**Comment by jlubeck** [comment]
Can we re open this please so it gets more attention? Thanks

**Comment by mickbyrne** [comment]
This is still a problem and no clearing of derived data is fixing the problem. Major issue and is preventing our releases. 

We need this fixed immediately.

**Comment by goharika** [comment]
I have got the same problem, did anyone fix it?

**Comment by mukeshthawani** [comment]
Hi @goharika can you try this: https://github.com/AppLozic/Applozic-iOS-Chat-Samples/tree/master/ApplozicCocoaPodDemo

See if you can reproduce it in the sample app so that we know what is creating the problem.

**Comment by mickbyrne** [comment]
I found that inside the AppLozic pod there's 3 folders with the same name. Specifically, in the `Applozic/Resources/` folder there's 3 folders named `en.lproj`. Confusing things further, two of the folders contain one file named `Localizable.strings`. Each folder corresponds to a different location on disk, but this seems to be the problem.

I worked around it by combining the strings in the 2 `Localizable.strings` file and then combining the contents into a single folder. 

This let me get my build out, but is not a feasible long term solution as it will happen again when the project is checked out by a different developer.
